rewrite-gradle/src/main/java/org/openrewrite/gradle/search/FindDependency.java

```diff
@@ -64,10 +63,35 @@ public String getDescription() {
 
     @Override
     public TreeVisitor<?, ExecutionContext> getVisitor() {
-        return new GradleDependency.Matcher()
+        GradleDependency.Matcher matcher = new GradleDependency.Matcher();
+        return matcher
```

Comment:
Perhaps I missed something, but this can just remain as is right?
```suggestion
        return new GradleDependency.Matcher()
```

Reply:
yes, reverted